### DISCUSSION TITLE
Возможность создания присетов персонажей

### DISCUSSION THREAD

**Discussion by KillerBunny031** [octothorp-team/public-dobrograd#732] [Предложения]
Было бы круто, если бы можно было создавать присеты своих персонажей. 
Ни у одного меня есть несколько персонажей, и, чтобы их менять мне каждый раз приходится открывать свой документ с сохраненными персонажами и копировать его на сервер. Это занимает много времени. Было бы круто, если можно было бы сменить перса нажатием одной кнопки.

**Comment by Kyper1** [comment]
Эта функция уже тестируется и в целом готова. Она выйдет в следующем крупном обновлении, где, по словам разработчиков, поменяется много всего.
